releasenotes/notes/branding-de80e7d9c1392e69.yaml

```diff
@@ -0,0 +1,9 @@
+---
+fixes:
+  - This release fixes a race condition where the
+    branding could be overriden depending on the playbook
+    execution order. Deployers have to make sure the
```

Comment:
@prometheanfire I don't think these are instructions that don't already appear in the proper location.